solutions/hashmaps/both-x.js

```diff
@@ -0,0 +1,50 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+    idx++;
+    return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+
+function bothCountX(string1, string2, x) {
+    // Complete this function, and return the list of resultant characters in sorted order
+    string1 = string1.toLowerCase();
+    string2 = string2.toLowerCase();
+    var dictionary1 = {};
+    for (let i = 0; i < string1.length; i++) {
+        if (string1[i] in dictionary1)
+            dictionary1[string1[i]] = dictionary1[string1[i]] + 1;
+
+        else
+            dictionary1[string1[i]] = 1;
+    }
+    var dictionary2 = {};
```

Comment:
change the variabale definition of dictionary2 to "let"

Reply:
@rpapavan97 done